CMakeLists.txt

```diff
@@ -16,6 +16,7 @@ option(ICUB_MODEL_GENERATE_SIMMECHANICS "Generate models using the model generat
 option(BUILD_TESTING "Run tests for the generated models" ON)
 
 set(BUILD_PREFIX "iCub")
+set(COMPONENTS_BUILD_PREFIX "iCub_components")
```

Comment:
There is any specific reason for not installing these in `iCub`  as the rest of the models? If we install the models in a different directory, then we need to:
* Update the docs of icub-models ( https://github.com/robotology/icub-models#by-installing-the-models )
* Update the setup scripts of the robotology-superbuild (see https://github.com/robotology/robotology-superbuild/blob/545ec06c6915e0581dfaa2a09cc435f37ad5d614/conda/multisheller/icub-models_deactivate.msh, and all the files in https://github.com/robotology/robotology-superbuild/tree/master/cmake/template)

While this can be done, it complicates maintenance, so probably we need to understand if it is worth it.